internal/db/schema/migrations/postgres/0/01_domain_types.up.sql

```diff
@@ -5,38 +5,38 @@ check(
   length(trim(value)) > 10
 );
 comment on domain wt_public_id is
-'Random ID generated with github.com/hashicorp/vault/sdk/helper/base62';
+'Random ID generated with github.com/hashicorp/shared-secure-libs/base62';
```

Comment:
It probably doesn't really matter, but this wont cause the comment to update during a migration

Reply:
Yeah, really not an issue!